src/logger.config.ts

```diff
@@ -0,0 +1,46 @@
+import * as winston from 'winston';
+
+export class LoggerConfig {
+  static initialized: boolean = false;
```

Comment:
eslint ta reclanado aqui, eu não sei se é melhor mudar a regra ou deixar que o TS faça inferencia do tipo do `initialized`.

<img width="942" alt="image" src="https://github.com/mitz-it/nest-log/assets/33238105/cef1f67b-423b-42be-91b6-17316a79db0f">

```suggestion
  static initialized = false;
```